test-data/samples/crawl2.py

```diff
@@ -0,0 +1,863 @@
+#!/usr/bin/env python3.4
+
+"""A simple web crawler."""
+
+# This is cloned from <asyncio>/examples/crawl.py,
+# with type annotations added (PEP 484).
+#
+# This version (crawl2.) has also been converted to use `async def` +
+# `await` (PEP 492).
```

Comment:
To make sure I understand things correctly: I thought you said they were necessary if you wanted the function to be usable in a `yield from`?